spec/controllers/tools_controller_spec.rb

```diff
@@ -48,9 +48,9 @@
 
         it 'creates a flash warning' do
           msg = 'You have passed a logout request. Currently, this tool is for ' +
-          'Authentication requests only. Please try this ' +
-          '<a href="https://www.samltool.com/validate_logout_req.php" target="_blank">' +
-          'tool</a> to validate logout requests.'
+                'Authentication requests only. Please try this ' +
+                '<a href="https://www.samltool.com/validate_logout_req.php" target="_blank">' +
+                'tool</a> to validate logout requests.'
```

Comment:
ditto for heredoc syntax?
```ruby
          msg = <<~EOS.squish
            You have passed a logout request. Currently, this tool is for
            Authentication requests only. Please try this
            <a href="https://www.samltool.com/validate_logout_req.php" target="_blank">
            tool</a> to validate logout requests
          EOS

Reply:
updated in https://github.com/18F/identity-dashboard/pull/776/commits/8726d8eefc76b44db9fdc5eb9c75e8ae25dba821